lib/array.js

```diff
@@ -22,7 +22,7 @@ const helpers = module.exports;
  */
 
 helpers.after = function(array, n) {
-  if (!Array.isArray(array)) return '';
+  if (utils.isUndefined(array)) return '';
```

Comment:
ok, but anyway `utils.isUndefined` should be added, because there are a lot of usages this, otherwise a set of helpers will be broken, for example https://github.com/oneflow/handlebars-kit/blob/master/lib/array.js#L585-L596

Reply:
Sorry, my comment was misleading, we also need to test that the argument `array` is present (otherwise it will throw an exception) and additionally test `array.slice` is a function.

Reply:
done

Reply:
I don't see the test for slice is a function, just that slice is a property.